game_logic/ball_holds/index.rst

```diff
@@ -40,7 +40,7 @@ the prefix for ball holds is ``device.ball_holds.<name>``.
 Related How To guides
 ---------------------
 
-.. todo:: :doc:`/about/help_us_to_write_it`
+:doc:`Using ball_holds for a mystery award </game_logic/mystery_award>`
```

Comment:
Warning, treated as error:

/home/travis/build/missionpinball/mpf-docs/game_logic/ball_holds/index.rst:43:unknown document: /game_logic/mystery_award

Reply:
Sorry about the errors, it was my first time creating a page for the docs, so I'm still learning the basics. Looks like the test passed after a couple of edits.